homeassistant/components/miele/sensor.py

```diff
@@ -42,11 +43,16 @@
 DISABLED_TEMPERATURE = -32768
 
 
+def _convert_duration(value_list: list[int]) -> int | None:
+    """Convert duration to minutes."""
+    return value_list[0] * 60 + value_list[1] if value_list else None
+
+
 @dataclass(frozen=True, kw_only=True)
 class MieleSensorDescription(SensorEntityDescription):
     """Class describing Miele sensor entities."""
 
-    value_fn: Callable[[MieleDevice], StateType]
+    value_fn: Callable[[MieleDevice], StateType | list[int]]
```

Comment:
```suggestion
    value_fn: Callable[[MieleDevice], StateType]
```
should not be necessary anymore